postreise/analyze/generation/summarize.py

```diff
@@ -18,11 +18,14 @@ def sum_generation_by_type_zone(
     """Get total generation for each generator type and load zone combination.
 
     :param powersimdata.scenario.scenario.Scenario scenario: scenario instance.
-    :param tuple time_range: [start_timestamp, end_timestamp] where each time stamp
+    :param tuple time_range: [start_timestamp, end_timestamp] where each timestamp
         is pandas.Timestamp/numpy.datetime64/datetime.datetime. If None, the entire
         time range is used for the given scenario.
-    :param str time_zone: new time zone.
+    :param str time_zone: new time zone. Note that
```

Comment:
`Note that` -> 🙅‍♂️ ?

Reply:
I forgot to delete it after moving the comment to the note.

Reply:
Done